docs/mining/proof-of-stake.md

```diff
@@ -1,22 +1,31 @@
-# <img class="dcr-icon" src="/img/dcr-icons/TicketVoted.svg" /> Proof-of-Stake (PoS) Mining
+# <img class="dcr-icon" src="/img/dcr-icons/TicketVoted.svg" /> Proof of Stake Ticket Voting
```

Comment:
This file refers to "Proof of Stake Ticket voting" whereas the other files simply refer to "Proof of Stake voting"

"Proof of Stake voting" is nicer I feel - adding ticket seems extraneous

Reply:
Agreed. Capitalized Voting to keep consistent with other titles,

# Proof of Stake Voting

Reply:
The header has been updated but the rest of the file still refers to "Ticket voting" - terminology not used anywhere else in the documentation

Reply:
Updated rest of file to use 'PoS voting'